src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
+    const progressNum = + ( ( mainStat / goal ) * 100 ).toFixed(2)
+    return ( progressNum ).toString().concat('%')
```

Comment:
A previous comment suggest not needing this line, but wanted to suggest that the use of parentheses around `progressNum` here makes this code look more complicated than it is, and serve no purpose other than to add characters in the code (they are not needed to increase precedence, for example).